build_debian.sh

```diff
@@ -260,6 +260,13 @@ sudo LANG=C chroot $FILESYSTEM_ROOT umount /proc
 ## Prepare empty directory to trigger mount move in initramfs-tools/mount_loop_root, implemented by patching
 sudo mkdir $FILESYSTEM_ROOT/host
 
+## Create loop device for /var/log to limit its size to 1000000*512=500MB
+sudo mkdir -p $FILESYSTEM_ROOT/var/disk-img
+sudo LANG=C chroot $FILESYSTEM_ROOT dd if=/dev/zero of=/var/disk-img/var-log.ext4 count=1000000
```

Comment:
I'm a bit confused about onie_platform.
Current build configuration requires only PLATFORM and SKU.
Aren't they the two keys that define a unique platform (in case of mellanox its enough)?
In that case conf file can be put under platform/$PLATFORM/sku/$SKU/

Do we have one image for different platforms in any vendor?